### DISCUSSION TITLE
Browser Console errors in generated html

### DISCUSSION THREAD

**Discussion by lukefullard** [quarto-dev/quarto-cli#2920] [Q&A]
Hi all,

Occasionally when I load a html page with inspect mode activated I see the following error in the console:

```
site_compare.html:3905 Uncaught Error: Mismatched anonymous define() module: [object Object]
https://requirejs.org/docs/errors.html#mismatch
    at makeError (site_compare.html:3905:1803)
    at T (site_compare.html:3905:8685)
    at site_compare.html:3905:15076
```

Also, sometimes Python generated figures don't show unless I refresh the page.

Has anyone had this issue before?

Thanks

**Comment by Soju-JC** [comment]
Same here, I have no idea what's going on. It is happenning with my plotly charts. I suspect the problem shows because of the 'hover' type arguments to customize the interactive traces in the charts, but need to do some more investigation.

**Reply by Sparkvni** [reply]
I've fixed it by removing `smooth-scroll: true` from the _quarto.yml

**Reply by Soju-JC** [reply]
> I've fixed it by removing `smooth-scroll: true` from the _quarto.yml

Nice, it's working now.

**Reply by mcanouil** [reply]
Thanks for the report, would any of you be able to provide a small Quarto document and the steps to reproduce this?

**Reply by Sparkvni** [reply]
Sure, here are the steps:


1- Create a Quarto website project. I did this using the most recent version of Vscode and Quarto.
2- In the `_quarto.yml` file, add the argument `smooth-scroll: true`. example:
```
format:
  html:
    smooth-scroll: true
```

3- Run the `plot.qmd` file inside the [plot.zip](https://github.com/quarto-dev/quarto-cli/files/11358564/plot.zip), (github won't let me upload a `.qmd` file), note that `Python 3.10` and `Plotly` are needed.
4- Inspect the preview in a browser and look for the error:


```
Uncaught Error: Mismatched anonymous define() module: [object Object]
https://requirejs.org/docs/errors.html#mismatch
    at makeError (require.min.js:1:1795)
    at T (require.min.js:1:8677)
    at require.min.js:1:15068
```

A couple of notes here: This problem only occurred when using Plotly. It's also worth mentioning that the speed of the website increased, although it's difficult to tell by how much.

I think that's all, let me know if you need anything else.

**Reply by mcanouil** [reply]
Thanks,
You can simply copy paste the content of the qmd in here, using extra back ticks as described in <https://quarto.org/bug-reports.html>.

**Reply by Sparkvni** [reply]
Oh sure thing, here is the content:

````
---
title: "plotly plot"
---
```{python}
import plotly.express as px
import plotly.io as pio
df = px.data.iris()
fig = px.scatter(df, x="sepal_width", y="sepal_length", 
                 color="species", 
                 marginal_y="violin", marginal_x="box", 
                 trendline="ols", template="simple_white")
fig.show()
```
````

**Reply by mcanouil** [reply]
Closing this discussion in favour of the opened issues to track this, see #5356.
